src/api/routes/edition.js

```diff
@@ -218,15 +254,109 @@ router.get('/:bbid/identifiers',
  *             $ref: '#/definitions/Relationships'
  *       404:
  *         description: Edition not found
- *       406:
+ *       400:
  *         description: Invalid BBID
  */
 
 router.get('/:bbid/relationships',
-	makeEntityLoader('Edition', relationshipsRelations, editionError),
-	async (req, res) => {
+	makeEntityLoader('Edition', utils.relationshipsRelations, editionError),
+	async (req, res, next) => {
 		const editionRelationshipList = await getEntityRelationships(res.locals.entity);
 		return res.status(200).send(editionRelationshipList);
 	});
 
+/**
+ *	@swagger
+ * '/edition':
+ *   get:
+ *     tags:
+ *       - Browse Requests
+ *     summary: Gets a list of Editions related to the other entity
+ *     description: BBID of an Author or an Edition or an EditionGroup or a Publisher or a Work is passed as query parameter and it's related Editions are fetched
+ *     operationId: getRelatedEditionByBbid
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: author/edition/edition-group/publisher/work
+ *         in: query
+ *         description: BBID of the corresponding entity
+ *         required: true
+ *         type: bbid
+ *     responses:
+ *       200:
+ *         description: List of Editions related to the other entity
+ *         schema:
+ *             $ref: '#/definitions/BrowsedEditions'
+ *       404:
+ *         description: author/edition/edition-group/publisher/work (other entity) not found
+ *       400:
+ *         description: Invalid BBID passed in the query params
+ */
+
+router.get('/',
+	formatQueryParameters(),
+	validateBrowseRequestQueryParameters(['author', 'edition', 'edition-group', 'work', 'publisher']),
+	makeEntityLoader(null, utils.relationshipsRelations, 'Entity not found', true),
+	loadEntityRelationshipsForBrowse(),
+	async (req, res, next) => {
+		function relationshipsFilterMethod(relatedEntity) {
+			if (req.query.format) {
+				const editionFormatMatched = _.toLower(relatedEntity.editionFormat) === _.toLower(req.query.format);
+				if (req.query.language) {
+					const editionLanguageMatched = relatedEntity.languages.includes(_.upperFirst(_.toLower(req.query.language)));
```

Comment:
This should filter by ISO 639 two- or three-letter code rather than by name. (I think)
Same below.